redis/src/sentinel.rs

```diff
@@ -769,4 +770,17 @@ impl SentinelClient {
         let client = self.async_get_client().await?;
```

Comment:
minor: this can call `get_async_connection_with_config` with default config

Reply:
Good thoughts! Did squash as well. N.B. I am not able to run all tests.